### ISSUE TITLE
Q: Smaller YCB objectset 

### ISSUE THREAD

**Issue by LarsJanssenTUe** [RoboCupAtHome/RuleBook#719]
Today we obtained some more information about how to obtain the YCB objectset. Below you can find the most important statements of the email we received:

"The cost of the set ($700) includes the cost of the objects and operational costs for labor to maintain our stock, source materials, manage and fulfill orders, etc."

"In order to process your order, please fill out the attached document and return to me, then I will follow up with a quote to confirm estimated total cost that includes estimated shipping. There will also likely be duties/customs/VAT costs, which can range between $50-450 depending on where it is shipping, which you will have to handle on your own."

"Please note that your kit may not ship for a few months, but once it does they are typically delivered in about a week."

In the following link: http://www.ycbbenchmarks.com/object-set/ it can be seen that there are 77 objects in the YCB objectset. From these objects we would use 13 objects according to https://github.com/RoboCupAtHome/AtHomeCommunityWiki/wiki/Standard-Objects.

This would mean that teams would, in the worst case, pay $1150 to obtain 13 objects.
@johaq (original credits to Leroy Rügemer) already forwarded a potential list of (similar) objects that could be obtained in the Telegram group. However he also indicated that most of these objects, at least to some extent, deviate from the original, which is undesirable.  
 
My question therefore is as follows:
Would it be an idea to approach the UMass Lowell NERVE Center, which handles the distribution of the YCB Object Sets in collaboration with the project organizers, as the RoboCupAtHome community to try and help each other out. If we, with a group of 25-30 teams (the teams participating in RGO and/or RWC) all buy an object set, they might be willing to make a selection of objects for a more reasonable price. 

TLDR: Let's combine forces to try and reduce the price of the object set!

**Comment by alex-mitrevski** [comment]
I completely agree; the whole set is too expensive to buy. It would be great if we could negotiate a subset of objects for a more reasonable price.

**Comment by fabricejumel** [comment]
I bought the complete dataset . Hoping  that in the future we will use it completely in robocup@home ;-) .
It took less than 3 weeks to receive it. 
Cost is important (total of 1000 euros for us ) but i personnaly like the idea of the YCB as a universal  grasping datasets
Idea of asking for an official subsets for all robocuper  is a  very good idea. 
Concerning questions about" datasets will be available during setup days of robocup." I hope that yes.

For OC and TC, how do you plan to have the datasets for each arena (and potentially a need of multiple pieces for objects) . In the past it was bought during setup days and pay by local oragniser, but seems more logical now logical to have 3 or more datasets proprety of robocup league ? 
Teams could probably lend some objects but they are really precious and very costly to replace . So not so simple to manage.

If people want to prepare training, they are welcome in our Lab in Lyon, France to proceed .

**Comment by RemiFabre** [comment]
Hi,
Any updates on this? Is the list of 13 items definitive? Will there be items that will be discovered during the setup days only?
We'll be going with the "alternative objects bought on Amazon" route for now too. 

We're willing to group up with you guys if we can buy the actual items with better conditions.

**Comment by LarsJanssenTUe** [comment]
> Hi,
> Any updates on this? Is the list of 13 items definitive? Will there be items that will be discovered during the setup days only?
> We'll be going with the "alternative objects bought on Amazon" route for now too.
> 
> We're willing to group up with you guys if we can buy the actual items with better conditions.

For now there is no update from me side. Since there was so little response we, as Tech United Eindhoven, don't see to much alternatives then buying the entire set. The "alternative objects bought on Amazon" for now do not seem like a good option to me.

**Comment by moriarty** [comment]
I recommend buying the whole set. It’s reusable and the plan is to enable high level capabilities by seeding the competition with known objects. If you’re in a lab which cares about publications outside of RoboCup, the use of the entire standardized object set provides comparable results.

The plan is to continue using this object set and eventually the cost per object will drop.
700$ / 77 objects ~= 9$/object for year 1. But these objects should last for multiple years.

RoboCup@Work has a standardized object set, of which the original set hasn’t changed, only new objects added. But the At-Work object set is much smaller, and only used within RoboCup / ERL. The YCB Dataset is large and diverse enough to continue to push the At-Home league.
